app/src/main/java/org/cru/godtools/viewmodel/GlobalDashboardDataModel.kt

```diff
@@ -3,12 +3,17 @@ package org.cru.godtools.viewmodel
 import android.app.Application
 import androidx.lifecycle.AndroidViewModel
 import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.map
+import org.ccci.gto.android.common.db.Query
+import org.cru.godtools.model.GlobalActivityAnalytics
+import org.keynote.godtools.android.db.GodToolsDao
 
-class GlobalDashboardDataModel(application: Application) : AndroidViewModel(application) {
+class GlobalDashboardDataModel(app: Application) : AndroidViewModel(app) {
 
-    // TODO:  Create temporary Data to be updated in later PR
-    val uniqueUsers = MutableLiveData<String>()
-    val gospelPresentation = MutableLiveData<String>()
-    val sessions = MutableLiveData<String>()
-    val countries = MutableLiveData<String>()
+    private val dao = GodToolsDao.getInstance(getApplication()).getLiveData(Query.select(GlobalActivityAnalytics::class.java).where("_id", 1))
```

Comment:
this isn't a `dao`, it's a specific live data query handle.

Reply:
also, there are better options for fetching a single record.

`dao.findLiveData(GlobalActivityAnalytics::class.java, 1)`